src/Template/Bake/Element/Controller/add.ctp

```diff
@@ -18,7 +18,7 @@ $compact = ["'" . $singularName . "'"];
     /**
      * Add method
      *
-     * @return void Redirects on successful add, renders view otherwise.
+     * @return \Cake\Network\Response|void Redirects on successful add, renders view otherwise.
```

Comment:
"Currently the bake doc block doesn't pass the cakephp standard, as redirect returns a response" sounds to me like this would fit perfectly into the scope.